library/src/G4CMPKaplanQP.cc

```diff
@@ -112,6 +115,8 @@ void G4CMPKaplanQP::SetFilmProperties(G4MaterialPropertiesTable* prop) {
     phononLifetime =      prop->GetConstProperty("phononLifetime");
     phononLifetimeSlope = prop->GetConstProperty("phononLifetimeSlope");
     vSound =              prop->GetConstProperty("vSound");
+    absorberEff =         prop->GetConstProperty("absorberEff");
```

Comment:
Implement these two assignment statements using the ternary operator (`: ?`) as is done with lowQPlimit.  Use a default of 1. for the efficiency, and 0. for the slope.